include/keepkey/firmware/coins.def

```diff
@@ -34,5 +34,6 @@ X(true, "GRS Testnet",  true, "tGRS",  true, 111,    true,     100000 , true, 19
 X(true, "Megacoin",     true, "MEC",   true,  50,    true,    1000000,  true,   5,  true, "\x19" "MegaCoin Signed Message:\n",      true, 0x800000d9,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,            false, "",            false, "",     false, false,  true,  77429938,        true, 78792518,            false, ""    )
 X(true, "Terracoin",    true, "TRC",   true,   0,    true,     100000,  true,   5,  true, "\x19" "DarkCoin Signed Message:\n",      true, 0x80000053,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,            false, "",            false, "",     false, false,  false,        0,        false,       0,            false, ""    )
 X(true, "Qtum",         true, "QTUM",  true,  58,    true,   40000000,  true,  50,  true, "\x15" "Qtum Signed Message:\n",          true, 0x800008fd,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, true,   true, false,  true, SECP256K1_STRING,            false, "",            true, "qc",    false, false,  true,  77429938,        true, 78792518,            false, ""    )
+X(true, "Hatch",        true, "HATCH", true,  76,    true,     100000,  true,  16,  true, "\x19" "Hatch Signed Message:\n",         true, 0x854c5638,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 50221772,  true, false,  true, false,  true, SECP256K1_STRING,            false, "",            false, "",     false, false,  false,        0,        false,       0,            false, ""    )
```

Comment:
```suggestion
X(true, "Hatch",        true, "HATCH", true,  76,    true,     100000,  true,  16,  true, "\x16" "Hatch Signed Message:\n",         true, 0x854c5638,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 50221772,  true, false,  true, false,  true, SECP256K1_STRING,            false, "",            false, "",     false, false,  false,        0,        false,       0,            false, ""    )
```

Reply:
If you are testing a wallet and therefore have made changes, then in the Hatch Core binaries that are now available on github BIP44 = 5, today we will compile new Hatch Core binaries with BIP44 = 88888888

Reply:
Good choice. Better not to use Dash's BIP44 paths.

Reply:
> `"\x16" "Hatch Signed Message:\n"`

This suggestion comes from counting the letters, since that first byte usually signifies the length of the prefix. If that's not correct for Hatch, you shouldn't apply it.

Reply:
Your changes are correct. Thank you for your contribution to the development of HATCH. At the moment, the HATCH repository contains the full version of Hatch Core with the latest changes in binary files, including the new BIP44.